src/bootstrap/dist.rs

```diff
@@ -669,20 +714,28 @@ pub fn extended(build: &Build, stage: u32, target: &str) {
         let _ = fs::remove_dir_all(&pkg);
         t!(fs::create_dir_all(pkg.join("rustc")));
         t!(fs::create_dir_all(pkg.join("cargo")));
+        t!(fs::create_dir_all(pkg.join("rls")));
+        t!(fs::create_dir_all(pkg.join("rust-analysis")));
```

Comment:
All of the changes in this file from here down only affect the platform-specific .msi and .pkg installers, and will not work correctly without further modifications of their definitions. I think you should remove them all for now (everything inside the `target.contains("apple-darwin")` and `target.contains("windows").